showcase/tests/integration/components/hds/app-header/home-link-test.js

```diff
@@ -3,59 +3,59 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import { module, test } from 'qunit';
+import { module /* , test */ } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, resetOnerror, setupOnerror } from '@ember/test-helpers';
-import { hbs } from 'ember-cli-htmlbars';
+// import { render, resetOnerror, setupOnerror } from '@ember/test-helpers';
+// import { hbs } from 'ember-cli-htmlbars';
 
 module('Integration | Component | hds/app-header/home-link', function (hooks) {
   setupRenderingTest(hooks);
 
-  hooks.afterEach(() => {
-    resetOnerror();
-  });
+  // hooks.afterEach(() => {
+  //   resetOnerror();
+  // });
 
-  test('it should render the component with a CSS class that matches the component name', async function (assert) {
-    await render(
-      hbs`<Hds::AppHeader::HomeLink @icon="hashicorp" @ariaLabel="HashiCorp" id="test-home-link" />`
-    );
-    assert.dom('#test-home-link').hasClass('hds-app-header__home-link');
-  });
+  // DISABLEtest('it should render the component with a CSS class that matches the component name', async function (assert) {
```

Comment:
suggestion: instead of introducing all these comments, I would prefer us to use `skip` (instead of `test`). it will make it easier to enable them with a single search and replace.

```suggestion
  skip('it should render the component with a CSS class that matches the component name', async function (assert) {
```

Reply:
Updated locally, thanks.